coverage.html

```diff
@@ -0,0 +1,5561 @@
+
+<!DOCTYPE html>
```

Comment:
we don't need this file. you can add this to gitignore

Reply:
@AnikaAgiwal2711 we have removed the file and added the coverage out for adding up more acceptance test